src/test/java/com/example/crudapi/integrationtest/LocationRestApiIntegrationTest.java

```diff
@@ -126,11 +126,11 @@ class CreateLocationTest {
                       "status": "400",
                       "error": "Bad Request",
                       "message": {
-                          "corner": "required item",
+                          "corner": "Please enter",
```

Comment:
目的語がないとenterは入る、登場するという意味になります。
なので目的語をいれましょう。

```suggestion
                          "corner": "Please enter any value",
```

Reply:
https://ejje.weblio.jp/content/enter
```
自動詞
1〔動詞(+前置詞+(代)名詞)〕〔…から〕入る 〔at，by，through〕《★この意味で enter into a room，house，train，etc. を用いるのは古い用法で，現在は他動詞の enter を用いる》.
enter at the door [through the window] ドア[窓]から入る.
2[Enter で] 【演劇】 登場する 《★脚本のト書きではしばしば 3 人称命令法で用いる; ⇔exit》.
```

Reply:
修正しました！ 95ea8f70c71c1dba0433054a1260c51fe1fc6a75